packages/response-ops/alerts_table/types.ts

```diff
@@ -331,6 +331,7 @@ export type RenderContext<AC extends AdditionalContext> = {
   showAlertStatusWithFlapping?: boolean;
 
   bulkActionsStore: [BulkActionsState, Dispatch<BulkActionsReducerAction>];
+  toggleColumn: (columnId: string) => void;
```

Comment:
One `nit` : It looks like toggleColumn is also available in `ref`, you may not need to change the context. But i leave it upto you.

https://github.com/elastic/kibana/pull/208052/files#diff-f0447f2afff0b21f2045c0e0ef43770c19183fb895af270dfe4c7498e6d537f0L370

Reply:
You're absolutely right, and since this overlaps with the already existing table ref I took the occasion and directly shared that ref instead which simplifies other things as well 🙂